### ISSUE TITLE
End Date can be Before Start Date

### ISSUE THREAD

**Issue by alocke8181** [umts/screaming-dinosaur#435]
I was working on moving some of the spec's from `controllers/assignments_controller_spec` to `requests/assignments_spec` and for one of them, I tried setting the end date to be before the start date. When it's submitted it appears to be successful. I also tried doing it by running the development server and submitting redirects to the home page with the success flash message.

Looking through everything, it appears only the `weekday_generator` model checks that the end date is after the start date. Could someone else take a look to make sure I'm not crazy?

**Comment by werebus** [comment]
Yeah, seems like you're right - we don't validate for it. Pretty easy to add, though, because Rails [has a built-in thing for that](https://guides.rubyonrails.org/active_record_validations.html#comparison).
